app/components/article-media-thumbnail.js

```diff
@@ -35,11 +33,15 @@ export default Component.extend(
 			return `${this.get('itemContext')}-${this.get('type')}`;
 		}),
 
+		viewportWidth: computed(() => {
+			return typeof Fastboot !== 'undefined' ? null : document.documentElement.clientWidth;
```

Comment:
Shouldn't we use some sane (more or less a mobile screen width) default for FastBoot?